samples/storage_csm_powerscale_v270.yaml

```diff
@@ -443,18 +443,24 @@ spec:
       configVersion: v1.6.0
       components:
         - name: podmon-controller
+          image: dellemc/podmon:v1.6.0
+          imagePullPolicy: IfNotPresent
           args:
             - "--labelvalue=csi-isilon"
             - "--arrayConnectivityPollRate=60"
             - "--skipArrayConnectionValidation=false"
             - "--driverPodLabelValue=dell-storage"
             - "--ignoreVolumelessPods=false"
+            - "--arrayConnectivityPollRate=5"
```

Comment:
@xuluna this parameter is defined 2 times, can you please check this.

Reply:
You are right. Fixed these in new PR: https://github.com/dell/csm-operator/pull/302

Reply:
Thanks.